src/app/layout.tsx

```diff
@@ -1,7 +1,7 @@
 import type { Metadata } from "next";
 import { Geist, Geist_Mono } from "next/font/google";
 import "./globals.css";
-
+import Header from "@/components/Header";
```

Comment:
other components have named export, let's keep the pattern going. We will add it as a lint enforcement pretty soon